SPECS/bpftrace/bpftrace.spec

```diff
@@ -51,6 +51,7 @@ cd build
 
 %cmake \
     -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS:BOOL=OFF \
```

Comment:
Should check if we really want to disable building shared libraries.

Reply:
Okay yup, going to disable shared library builds so we don't generate unnecessary shared libraries. We can create a subpackage for the libraries if we start having apps that need to link against a libbpftrace.so.

The bpftrace app itself is still dynamically linked so we're good there.